package.json

```diff
@@ -48,7 +48,8 @@
     "rollup-plugin-terser": "^7.0.2",
     "sass": "^1.58.3",
     "tailwindcss": "^3.2.7",
-    "tiny-emitter": "^2.1.0"
+    "tiny-emitter": "^2.1.0",
+    "wouter-preact": "^2.10.0-alpha.1"
```

Comment:
This is the latest stable version of the Preact variant of `wouter`. I think the `-alpha.1` suffix is an oversight, but I'll check.